### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by Laranjo1** [vfsfitvnm/frida-il2cpp-bridge#440]
I have some questions to ask, I'm new to the subject so I might end up asking stupid and meaningless things.

First of all, is there any tutorial on how to configure it for Windows? I had managed to compile and assemble a build, but when it spawned, the bridge was not allocated, only the console outside the bridge structure.

Another issue I have is that when I use frida-ps -U
I get "The Game Arena" as my game's process name. Can I run the spawn for it? If so, why does it recognize this process name and not com.gamedragonball in frida-ps -U?

Third is the least related to this repository but I was curious to know what it could be, I apologize. Well, my doubt is, because when I allocated the intreceptor in a method of the class it recognized that the function was executed, but when I tried to see the parameters I couldn't get the values ​​that came in the parameters that were int. What could I be doing wrong?

Can I use an emulator to debug il2cpp game? If so, how could I configure it for this purpose? I know there is a parameter for this, is that enough?

Sorry for my english prob is bad but max i can write mb

**Comment by leonitousconforti** [comment]
> First of all, is there any tutorial on how to configure it for Windows? I had managed to compile and assemble a build, but when it spawned, the bridge was not allocated, only the console outside the bridge structure.

I don't think so, but frida-il2cpp-bridge doesn't really have anything to configure as it is just an npm package you install. Are you trying to setup/configure frida for windows?

> Another issue I have is that when I use frida-ps -U I get "The Game Arena" as my game's process name. Can I run the spawn for it? If so, why does it recognize this process name and not com.gamedragonball in frida-ps -U?

What platform are you on when you see that? If it is an android/ios app you should be able to spawn it with `frida -U -f com.gamedragonball` but not sure if you can do that on windows/mac

> Third is the least related to this repository but I was curious to know what it could be, I apologize. Well, my doubt is, because when I allocated the intreceptor in a method of the class it recognized that the function was executed, but when I tried to see the parameters I couldn't get the values ​​that came in the parameters that were int. What could I be doing wrong?

Not sure, can you share your code with us?

> Can I use an emulator to debug il2cpp game? If so, how could I configure it for this purpose? I know there is a parameter for this, is that enough?

If you mean an android emulator? Yes, I use frida-il2cpp-bridge with the android emulator that comes with android studio all the time. I install frida-server on the device and then install the apk I want to debug. I haven't done an ios app in a while but the process is similar

**Comment by Laranjo1** [comment]
>> I don't think so, but frida-il2cpp-bridge doesn't really have anything to configure as it is just an npm package you install. Are you trying to setup/configure frida for windows?

Yes, could you send the necessary commands for the correct installation on Windows to debug an Android game on the emulator? 

>> What platform are you on when you see that? If it is an android/ios app you should be able to spawn it with frida -U -f com.gamedragonball but not sure if you can do that on windows/mac

Im using android, it spawned but the script didn't load in the game i really idk why

>> code

```js
function awaitForCondition(callback) {
    var i = setInterval(function () {
        var addr = Module.findBaseAddress('libil2cpp.so');
        //// console.log("Address found:", addr);
        if (addr) {
            clearInterval(i);
            callback(+addr);
        }
    }, 0);
}
Java.perform(function () {
    awaitForCondition(function (base) {
        il2cpp = ptr(base);
        PlayerID()
    })
})

function playerID() {
    Interceptor.attach(il2cpp.add("0x828171"), {
        onEnter: function (args) {
            Console.log(agrs[0]);
        }
    })
}


```


>> If you mean an android emulator? Yes, I use frida-il2cpp-bridge with the android emulator that comes with android studio all the time. I install frida-server on the device and then install the apk I want to debug. I haven't done an ios app in a while but the process is similar

Ok ty i go try install that emulator.
Its better if using emulator.

Edit: need emulate arm64 processor?


Edit2: 
![imagem](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/91355461/29b44b7c-0cd0-4561-9a29-ae2a7a03e6bd)

am I missing something?

Edit again: Im a stupid guy, Im trying Run Frida server on android 9 emulated XD

**Comment by Laranjo1** [comment]
I can send the game I'm using to test if you're interested

**Comment by Laranjo1** [comment]
I installed the bluestacks5 emulator and in the ABI options I only selected arm64 but it still shows that the processor is x86_64
![image](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/91355461/b0c89be3-af3e-4e51-8fd0-4dc1852f6f65)
